app/data_models/data_stores.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass, field
+from typing import MutableMapping
+
+from app.data_models.answer_store import AnswerStore
+from app.data_models.list_store import ListStore
+from app.data_models.metadata_proxy import MetadataProxy
+from app.data_models.progress_store import ProgressStore
+from app.data_models.supplementary_data_store import SupplementaryDataStore
+
+
+@dataclass
+class DataStores:
+    # self.metadata is a read-only view over self._metadata
+    metadata: MetadataProxy | None = None
```

Comment:
Non blocker - this comment doesn't apply any more, either we need `_metadata` here or reference questionnaire store

Reply:
@katie-gardner not sure about the best wording for the final comment, can it be `# self.metadata is a read-only view over QuestionnaireStore's self._metadata` ?

Reply:
It's something from before my time on the team so I never looked into details of this. If we add a setter, it will cause all these issues we discussed before.

Reply:
I don't think the comment is needed, otherwise it would be needed everywhere we pass `MetadataProxy`. It also is unclear as it is reference another class' property. We could get rid of that property and keep unmodified version as part of MetadataProxy, it's not uncommon for a class to have "_raw_data". It means 2 classes don't need to store the same thing.

Reply:
Removed comment.